game-runner/src/main/java/za/co/entelect/challenge/botrunners/BotRunnerFactory.java

```diff
@@ -28,6 +28,8 @@ public static BotRunner createBotRunner(BotMetaData botMetaData, int timeoutInMi
                 return new HaskellBotRunner(botMetaData, timeoutInMilliseconds);
             case PHP:
                 return new PHPBotRunner(botMetaData, timeoutInMilliseconds);
+            case SCALA:
+                return new ScalaBotRunner(botMetaData, timeoutInMilliseconds);
```

Comment:
In the BotLanguage.java file, you need to add SCALA as a language type, with this line:
```
    @SerializedName("scala")
    SCALA,
```